src/comparisons/events/on/ie9.js

```diff
@@ -1,15 +1 @@
-function addEventListener(el, eventName, eventHandler, selector) {
-  if (selector) {
-    el.addEventListener(eventName, function (e) {
-      if (e.target && e.target.matches(selector)) {
-        eventHandler(e);
-      }
-    });
-  } else {
-    el.addEventListener(eventName, eventHandler);
-  }
-}
-
-addEventListener(el, eventName, eventHandler);
-// Or when you want to delegate event handling
-addEventListener(el, eventName, eventHandler, selector);
+el.addEventListener(eventName, eventHandler);
```

Comment:
are we sure this is right for IE9?  This looks more like es6.js -- wrong file edited?

One of the things that doesn't make this a drop-in for JQuery is that there could be multiple elements in the JQuery selector, so a more JQuery drop-in replacement would be something like:

```
for (const el of document.querySelectorAll(selector)) {
    el.addEventListener(eventName, eventHandler);
}
```

Reply:
It depends on whether we want to treat the jQuery selector as selecting a single element or multiple for the examples. Virtually every other example assumes that only a single element is selected.

Reply:
This is a little different--the selector is an optional handler for child elements of the given element. So, for example, `$(el).on('click', handler, 'child-selector');` only handles events that match the given `child-selector` of the element `el`. The original implementation is probably the most accurate here